developer-docs-site/docs/tutorials/your-first-nft-sdk.md

```diff
@@ -0,0 +1,376 @@
+Alice's token balance: 1
+Bob's token balance: 0
+```
+
+This example demonstrates:
+
+* Initializing the REST and Faucet clients
+* The creation of two accounts: Alice and Bob
+* The funding and creation of Alice and Bob's accounts
+* The creation of a collection and a token using Alice's account
+* Alice offering a token and Bob claiming it
+* Bob unilaterally sending the token to Alice via a multiagent transaction
+
+## Step 4: The SDK in Depth
+
+### Step 4.1: Initializing the Clients
+
+In the first step, the example initializes both the REST and Faucet clients. The REST client interacts with the REST API, whereas the Faucet client is a devnet
+ service for creating and funding accounts.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```python
+:!: static/sdks/python/examples/simple-nft.py section_1
+```
+
+`common.py` initializes these values as such:
+```python
+:!: static/sdks/python/examples/common.py section_1
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+:::tip
+
+The URLs for both services, by default, point to our devnet services, however, they can be configured via setting following environmental variables: `APTOS_NOD
+E_URL` and `APTOS_FAUCET_URL`.
+
+:::
+
+
+### Step 4.2: Creating local accounts
+
+The next step is to create two accounts from the locally. [Accounts][account_basics] represent both on and off-chain state. The off-chain state consists of an
+address and the public, private key pair used to authenticate ownership. This step demonstrates how to generate that off-chain state.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```python
+:!: static/sdks/python/examples/simple-nft.py section_2
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+### Step 4.3: Creating blockchain accounts
+
+In Aptos, each account must have an on-chain representation in order to support receive tokens, coins, and interacting in other DApps. In Aptos, an account rep
+resents a medium for storing assets, hence it must be explicitly created. This example leverages the Faucet to create and fund Alice's account and to just crea
+te Bob's:
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```python
+:!: static/sdks/python/examples/simple-nft.py section_3
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+### Step 4.4: Creating a collection
+
+Now begins the process of creating tokens. First, the creator must create a collection to store the tokens in. A collection can contain 0, 1, or many distinct tokens within it. The collection does not restrict the attributes of the tokens, it is only a container.
```

Comment:
`the creator must create a collection to store tokens` no need for `the` or `in`

Nit: preferred when using `0, 1, many` to `zero, one, many`

I think you need `as` since you're basically putting 2 sentences together with a comma.
`restrict the attributes of the tokens, as it is only a container`